script/test_whiten.py

```diff
@@ -3,7 +3,8 @@
 import scipy
 from scipy.fftpack import next_fast_len
 
-from noisepy.seis.noise_module import moving_ave, whiten
+from src.noisepy.seis.datatypes import FreqNorm
+from src.noisepy.seis.noise_module import moving_ave, whiten
```

Comment:
hmm, the `src.` prefix shouldn't be required. Did you `pip install -e .` as described in the CONTRIBUTING.md file?

Reply:
Done.